### PR TITLE
Ruby on Rails Capstone Project: Touha_budget_app

### PR DESCRIPTION
**Pull request by TahenyBELGUITH**
**Budget Manager**

In this project I did the following:
 - [x] Setup the project with `rails`, `devise`, `rspec`, `bootstrap`, `linters`...
 - [x] Created models and database schema based on the entity relationship diagram provided in the requirements.
 - [x] Setup authentication using `devise`.
 - [x] Created the controllers and routes.
 - [x] Added views and styled them with bootstrap.
 - [x] Added model tests and feature tests.

<hr>

**Presentation video:**
Please check out the video [here](https://www.loom.com/share/8482fbc497b844f7b0a62624945e869d)

<hr>

Thank you for the review and happy coding 😊👋

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## Live demo:
+You can check out the live demo [here](https://touhax.herokuapp.com/).
+
+![316766130_5884636591628382_6427041912695299858_n_AdobeExpress](https://user-images.githubusercontent.com/57154534/203571050-532d9a5f-02ac-445c-8864-b272623e5262.gif)
+
+
+## Entity Relationship Diagram
+<img src="https://user-images.githubusercontent.com/50721479/190148367-be9ac1c0-2d96-4049-aea3-8e5d75c1f5fa.png" width="600">
+
```
Comment (Davidosky007) [inline README.md:15]: * Kindly update your readme file by using the [current readme template](https://github.com/microverseinc/readme-template)

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  margin: auto;
+}
+
+p {
+  font-size: 13px;
+}
+
+.navbar-dark .navbar-brand,
+.navbar-dark .navbar-toggler {
+  color: #ffc107;
```
Comment (Davidosky007) [inline app/assets/stylesheets/application.css:42]: * Kindly follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) as stated in [the requirements](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=You%20should%20follow,space%20between%20elements.) , including:

> Colors.
> Typography: font face, size and weight.
> Layout: composition and space between elements.

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    t.index ["user_id"], name: "index_groups_on_user_id"
+  end
+
+  create_table "spendings", force: :cascade do |t|
+    t.string "name"
+    t.integer "amount"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "group_id", null: false
+    t.bigint "user_id", null: false
```
Comment (Davidosky007) [inline db/schema.rb]: * do not change column names visible in the diagram check project [requirement for reference](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=do%20not%20change%20column%20names%20visible%20in%20the%20diagram.%20You%20need%20to%20change%20the%20%22Entity%22%20name%20according%20to%20the%20theme%20you%20have%20chosen%20for%20your%20project%20(please%20note%3A%20%22Transaction%22%20is%20a%20name%20already%20used%20by%20ActiveRecord%2C%20so%20using%20it%20as%20a%20name%20for%20your%20model%20and%20table%20will%20result%20in%20an%20error).) . Kindly stick to the column names as seen in the ERD diagram 🙏

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @TahenyBELGUITH 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* Professional looking video presentation 👍 
* Well written PR description 👍 
* Descriptive commit messages 👍 
* No linter errors 👍 
* All tests passed ✔️  

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by descholar-ceo — APPROVED** [review]
Hi @TahenyBELGUITH 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Professional looking video presentation ✔️ 
👍🏽 Well written PR description ✔️ 
👍🏽 Descriptive commit messages ✔️ 
👍🏽 No linter errors ✔️ 
👍🏽  All tests passed :heavy_check_mark:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**